scripts/stage-docs.sh

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+set -e
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+. "${SCRIPT_DIR}/common.sh"
+# Tmp fix - revert the nounset change made by common to match the expectations of this script.
+set +u
+
+REPOSITORY="origin"
+BRANCH_FROM="main"
+DRY_RUN="false"
+ORIGINAL_GH_DEFAULT_REPO=""
+while getopts ":v:u:b:r:dh" opt; do
+  case $opt in
+    v) RELEASE_VERSION="${OPTARG}"
+    ;;
+    u) WEBSITE_REPO_URL="${OPTARG}"
+    ;;
+    b) BRANCH_FROM="${OPTARG}"
+    ;;
+    r) REPOSITORY="${OPTARG}"
```

Comment:
You are clone the kroxylicious.github.io repo, so the upstream will always be origin.   I think this is superfluous.

Reply:
Agreed, this was a carry-over from the parts of `stage-release.sh` that I borrowed. I've removed it.